stable/pipelines/values.yaml

```diff
@@ -48,9 +89,9 @@ pipelines:
   # version:
 
   ## Artifactory URL - Mandatory
-  jfrogUrl:
+  jfrogUrl: ""
   ## Artifactory UI URL - Mandatory
-  jfrogUrlUI:
+  jfrogUrlUI: ""
```

Comment:
I suggest adding an example of what might be `jfrogUrlUI` compared to `jfrogUrl`. I don't know what is the difference.

Reply:
Done